### ISSUE TITLE
serve logging should use unscaped URIs

### ISSUE THREAD

**Issue by pyhedgehog** [rclone/rclone#7370]
#### The associated forum post URL from `https://forum.rclone.org`

N/A

#### What is your current rclone version (output from `rclone version`)?

rclone v1.63.1

#### What problem are you are trying to solve?

Logging of `rclone serve dlna` looks very misleading if there are non-latin filenames:
```
2023/10/14 03:56:31 INFO  : /r/%D0%91%D0%B0%D1%80%D0%B4%D1%8B/%D0%90%D0%B4%D1%80%D0%B8%D0%B0%D0%BD%20%D0%B8%20%D0%90%D0%BB%D0%B5%D0
%BA%D1%81%D0%B0%D0%BD%D0%B4%D1%80/2009%20Para%20Bellum/08%20%D0%9F%D0%B0%D0%B2%D0%B5%D0%BB%D0%B5%D1%86%D0%BA%D0%B8%D0%B9%20%D0%B2%D
0%BE%D0%BA%D0%B7%D0%B0%D0%BB%20%28%D1%81%D1%82%D0%B8%D1%85%D0%B8%29.mp3: 192.168.1.77:46742 GET 200
```
It will be much better to see something like this:
```
2023/10/14 03:56:31 INFO  : /r/Барды/Адриан и Александр/2009 Para Bellum/08 Павелецкий вокзал (стихи).mp3: 192.168.1.77:46742 GET 200
```

#### How do you think rclone should be changed to solve that?

I'm not sure that this should be forced for all cases, but at least we should have such option (with default to human-readable form).
So, I think that here:
https://github.com/rclone/rclone/blob/b9727cc6ab7ca563c5bb7fe83c00d1fddc1a0feb/cmd/serve/dlna/dlna_util.go#L111-L113
`lrw.request.URL` should be changed to `lrw.request.URL.Path` with option (something like `--dlna-log-full-url`) to revert.

**Comment by ncw** [comment]
Yes this looks like a sensible change. Do you want to send a pull request?

**Comment by ncw** [comment]
I don't think there is a need for a flag.

**Comment by pyhedgehog** [comment]
I'm not golang pro, so I can only send patch without flag.
There are two situations where flag can be important:

1. To debug correct decoding (i.e. `/%2f/` will look like `///`).
2. To debug query-string (which is not included in URL.Path).

**Comment by ncw** [comment]
I've merged @hsdfat this to master now which means it will be in [the latest beta](https://beta.rclone.org/) in 15-30 minutes and released in v1.76
